packages/create-sitecore-jss/src/templates/nextjs/src/byoc/index.client.tsx

```diff
@@ -8,6 +8,8 @@ import * as FEAAS from '@sitecore-feaas/clientside/react';
 // SitecoreForm component which enables new Sitecore Forms for XMCloud functionality. Requires a license to be available in Pages.
 // Sitecore Forms for Sitecore XP are still available separately via @sitecore-jss-forms package
 import '@sitecore/components/form';
+import React from 'react';
+import { ComponentProps } from 'react';
```

Comment:
Should we move these to the top to disassociate them from the component imports?

Reply:
yes, good idea